tox.ini

```diff
@@ -6,10 +6,19 @@ envlist = py37,py38,py39,py310,py311
 passenv = LOGNAME,USER,LNAME,USERNAME
 # https://packaging.python.org/guides/index-mirrors-and-caches/#caching-with-pip
 install_command=python -m pip install --disable-pip-version-check --find-links=cache {opts} {packages}
-commands = python -m unittest discover -s src {posargs}
+commands = python -m unittest discover --failfast -s src {posargs}
```

Comment:
Inclined to move `--failfast` to the the makefile, under the `tox_quick` target.

SHould also come in here via `posargs`.